source/index.txt

```diff
@@ -32,145 +38,297 @@
+                         "start": { "bsonType": "date" },
+                         "end": { "bsonType": "date" },
+                         "location": { "bsonType": "string" }
+                       }
+                     }
+
+               .. tab:: GraphQL Schema
+                  :tabid: graphql
+
+                  .. code-block:: graphql
+
+                     type Meeting {
+                       _id: ObjectId
+                       start: DateTime!
+                       end: DateTime!
+                       location: String!
+                     }
+
+         .. step:: Secure Your API
+            
+            Generate an API key that you can use to authenticate
+            requests. Alternatively, you can use other authentication
+            provider credentials associated with your app.
+            
+            For details, see :ref:`GraphQL API Authentication <graphql-authentication>`.
+
+         .. step:: Send Your First Request
+
+            Send your first GraphQL request from any server-side or
+            frontend environment that supports HTTPS. You can also use a
+            specialized GraphQL request tool or library.
+            
+            Learn how to :realm:`Send GraphQL Requests with Apollo
+            Client </web/graphql-apollo-react/>`.
+
+      .. image:: /images/plp/connection.png
+         :alt: Build a Secure GraphQL API
+
+   .. tab:: Build Event-Driven Services
+      :tabid: triggers
       
-      Learn more about the problems {+service+} solves - and how - to 
-      decide if App Services fits your use case.
+      .. procedure::
+
+         .. step:: Create and configure your Trigger
+
+            You can respond to database CRUD events and application user
+            activity as the occur or run triggers on a schedule. You can
+            also set additional conditions for when your trigger can
+            run. For example, you can set a database trigger to run only
+            for events where a document's date property was updated.
+
+         .. step:: Write the Trigger's Handler Logic
 
-Explore {+service+}
---------------------------
+            Define what happens when the trigger runs in response to an
+            event. The trigger can pass data about each event to AWS
+            Eventbridge or a serverless JavaScript function that you
+            write. You can import dependencies from npm and write custom
+            code to do anything you want to.
+            
+            See :doc:`Trigger Function Examples </triggers/examples>`.
+
+         .. step:: Automatically Respond to Events
+
+            Once deployed, the trigger automatically listens for events
+            in your app and runs whenever an event matches the trigger’s
+            configuration.
+
+      .. image:: /images/plp/triggers.png
+         :alt: Respond to Events with Triggers
+
+.. kicker:: Power Your App
+
+Run And Secure Your Services With A Fully-Managed Application Server
```

Comment:
When I saw this on the staged version it read pretty long.. any thoughts on 

"Run your services with additional backend features"